totalRP3/Modules/Dashboard/WhatsNewTabView.lua

```diff
@@ -16,19 +16,121 @@ local UITooltip = TRP3_API.ui.tooltip;
 local loc = TRP3_API.loc;
 local strhtml = TRP3_API.utils.str.toHTML;
 
---- List of "What's new" segments to concatenate and display together when
----  this view is activated.
-local SEGMENTS = {
-	loc.WHATS_NEW_28_4,
-	loc.WHATS_NEW_28_3,
-	loc.WHATS_NEW_28_2,
-	loc.WHATS_NEW_28_1,
-	loc.WHATS_NEW_28_0,
-};
+local CHANGELOG_TEXT = [[
+# Changelog version 3.0.0
+
+
+This version adds support for patch 11.0.
+
+Thanks to {col:00ff00}Raenore{/col} for the help on some of these features, and {col:00ff00}keyboardturner{/col} for the new silhouette logo.
+
+### {col:FFD200}For a visual guide to the major changes of this version, please see [this article](https://github.com/Total-RP/Total-RP-3/wiki/Guide-to-version-3.0) on our wiki.{/col}
+
+
+## Added
+
+
+- Added a new Relations menu located in the Customization section, alongside the moved Automation settings.
+  - This menu lets you edit the preset relations you can have with other players, as well as create new ones.
+  - Those relations are still only personal, and not shared with other players.
+  - The relation will now be shown as a line on the tooltip in addition to the border color. This can be changed to show the relation description, or disabled in Tooltip settings.
+- Added a "Currently" toolbar button letting you open a frame to quickly modify your Currently info.
+- Added a toolbar button to open/close the main Total RP 3 window.
+- Added a target frame button for your own companion to auto-fill the speech window with its name in brackets.
+  - This will apply RP name customization to its speech lines.
+- Added a dropdown to set whether you are open to other players walking up to your character.
+  - This will show as an icon on your tooltip and in your profile characteristics.
+  - Existing tooltip bottom icons have also been changed for clarity.
+    - Book: Profile has an unread description.
+    - Note: You have notes on this profile.
+    - Three people: Walkup friendly.
+- Added the new unread description icon to the About tab when relevant.
+- Added a tooltip setting to hide the addon version text and trial indicator separately from the bottom icons.
+- Added the ability to drag and drop personality traits to reorder them
+  - Additional info now uses the same button style rather than drag and dropping the icon itself.
+- Added the ability to duplicate a personality trait or additional info line, as well as convert preset lines into custom.
+  - These are located within the new Options button at the end of the line, alongside deletion of the line.
+  - Preset additional info lines appearing on the tooltip will no longer appear if converted to custom.
```

Comment:
I feel as though these points can be moved up a bit - at least above the "hide addon version text" note. Mostly because this is a cool set of features and wouldn't want them too far down.